foundations/likvid-prod/platforms/azure/meshplatform/terragrunt.hcl

```diff
@@ -46,6 +46,10 @@ inputs = {
     dependency.organization_hierarchy.outputs.landingzones_id
   ]
 
+  can_delete_rgs_in_scopes = [
+    "/subscriptions/e1c85eff-0a2c-4268-9b6c-7d2ff9ca9848"
```

Comment:
I'm not sure if that's possible here but if we could somehow link that subscription id to some other platform module that's configuring it (e.g. i think it's the "Lift & Shift LZ" that is set up in that Subscription)

Reply:
done in https://github.com/likvid-bank/likvid-cloudfoundation/pull/136/commits/39097d31888754d54aba3e5b76941cf523f53fa6